From dc369efc60206a6f0c23482fd27aef1e7360b5e8 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Thu, 13 Nov 2008 15:35:08 -0800
Subject: Staging: go7007: fix minor build warnings

Comment out some functions that we aren't using yet, in anticipation
of using them when the v4l core merges are changed upstream.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/go7007/go7007-v4l2.c |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

--- a/drivers/staging/go7007/go7007-v4l2.c
+++ b/drivers/staging/go7007/go7007-v4l2.c
@@ -327,6 +327,7 @@ static int set_capture_size(struct go700
 	return 0;
 }
 
+#if 0
 static int clip_to_modet_map(struct go7007 *go, int region,
 		struct v4l2_clip *clip_list)
 {
@@ -599,6 +600,7 @@ static int mpeg_g_control(struct v4l2_co
 	}
 	return 0;
 }
+#endif
 
 static int vidioc_querycap(struct file *file, void  *priv,
 					struct v4l2_capability *cap)
@@ -1089,7 +1091,7 @@ static int vidioc_s_parm(struct file *fi
 
    The two functions bellow implements the newer ioctls
 */
-
+#if 0
 static int vidioc_enum_framesizes(struct file *filp, void *priv,
 				  struct v4l2_frmsizeenum *fsize)
 {
@@ -1131,6 +1133,7 @@ static int vidioc_enum_frameintervals(st
 
 	return 0;
 }
+#endif
 
 static int vidioc_s_std(struct file *file, void *priv, v4l2_std_id *std)
 {
